types/chrome-remote-interface/index.d.ts

```diff
@@ -197,12 +197,16 @@ declare namespace CDP {
         Tracing: ProtocolProxyApi.TracingApi;
         WebAudio: ProtocolProxyApi.WebAudioApi;
         WebAuthn: ProtocolProxyApi.WebAuthnApi;
-    } & EventPromises<ProtocolMappingApi.Events>;
+    } & EventPromises<ProtocolMappingApi.Events> & EventCallbacks<ProtocolMappingApi.Events>;
 
     // '<domain>.<event>' i.e. Page.loadEventFired
     type EventPromises<T extends ProtocolMappingApi.Events> = {
         [Property in keyof T]: () => T[Property] extends [any] ? Promise<T[Property][0]> : Promise<void>;
     };
+
+    type EventCallbacks<T extends ProtocolMappingApi.Events> = {
+        [Property in keyof T]: (callback: (params: T[Property] extends [any] ? T[Property][0] : undefined, sessionId?: string) => void) => () => void;
```

Comment:
Based on https://github.com/cyrus-and/chrome-remote-interface/blob/v0.31.0/lib/api.js#L49 and because the anonymous function there has no braces, technically it should be:

```suggestion
        [Property in keyof T]: (callback: (params: T[Property] extends [any] ? T[Property][0] : undefined, sessionId?: string) => void) => () => NodeJS.EventEmitter;
```

as per https://nodejs.org/api/events.html#events_emitter_removelistener_eventname_listener, but exposing that seems to be more confusing than useful so `void` is fine with me.

Reply:
Probably even more accurate is:

```suggestion
        [Property in keyof T]: (callback: (params: T[Property] extends [any] ? T[Property][0] : undefined, sessionId?: string) => void) => () => CDP.Client;
```

but still `void` seems to be preferable.